src/utils.ts

```diff
@@ -41,27 +38,9 @@ export function sleepExact(period: number) {
   });
 }
 
-export function readBigUInt64LE(buffer: Buffer, inputOffset = 0) {
-  let offset = inputOffset;
-  const first = buffer[offset];
-  const last = buffer[offset + 7];
-  if (first === undefined || last === undefined) {
-    throw new Error();
-  }
-  const lo = first
-    + buffer[++offset] * 2 ** 8
-    + buffer[++offset] * 2 ** 16
-    + buffer[++offset] * 2 ** 24;
-  const hi = buffer[++offset]
-    + buffer[++offset] * 2 ** 8
-    + buffer[++offset] * 2 ** 16
-    + last * 2 ** 24;
-  return BigInt(lo) + (BigInt(hi) << BigInt(32));
-}
-
 export function parseDipState(buf: Buffer) {
-  const strike = Number(readBigUInt64LE(buf, 8));
-  const expiration = Number(readBigUInt64LE(buf, 16));
+  const strike = Number(buf.readBigUInt64LE(8));
```

Comment:
Where does this usage get imported?

Reply:
It is on buffer which is in typescript

Reply:
https://www.geeksforgeeks.org/node-js-buffer-readbiguint64le-method/